book_source/04_advanced_user_guide/02_adding_to_pecan/01_case_studies/04_input_conversion.Rmd

```diff
@@ -237,6 +237,56 @@ With the input id  in hand you can now edit a pecan XML so that the PEcAn functi
     </inputs>
 ```
 
+This IC workflow also supports generating ensembles of initial conditions from posterior estimates of DBH. To do this the tags below can be inserted to the pecan.xml:
+```
+       <css>
+        <source>PalEON</source>
+        <output>css</output>
+        <source.id>1000015682</source.id>
+        <useic>TRUE</useic>
+        <ensemble>20</ensemble>
+        <metadata>
+          <area>1256.637</area>
+          <n.patch>3</n.patch>
+        </metadata>
+      </css>
+```
+Here the `source.id` should point to a file that has MCMC samples to generate the ensemble from. The number between the `<ensemble>` tag defines the number of ensembles requested. The workflow will populate the settings list `run$inputs` tag with ensemble member information. E.g.:
```

Comment:
In all other contexts, the input ID referenced within a specific in put is just called `<id>`. In the next version I think we should revert to that here, rather than deviating from that norm, once the veg IC workflow is capable of distinguishing raw data, pecan standard, and model-specific file IDs (which should be discernable from the format ID associated with that input ID)